packages/react-magma-dom/src/components/Checkbox/Checkbox.stories.tsx

```diff
@@ -15,69 +12,135 @@ export default {
+          onChange={() => updateChecked(!checked)}
+        />
+      </FormGroup>
+      <FormGroup labelText="Colors">
+        <Checkbox
+          checked
+          color={magma.colors.primary}
+          labelText="Primary checked"
+          onChange={() => updateChecked(!checked)}
+        />
+        <Checkbox
+          checked
+          color={magma.colors.success}
+          labelText="Success checked"
+          onChange={() => updateChecked(!checked)}
+        />
 
-  function handleUpdateRedChecked(event: React.ChangeEvent<HTMLInputElement>) {
-    setCheckedItems([event.target.checked, checkedItems[1]]);
-  }
+        <Checkbox
+          checked
+          color={magma.colors.danger}
+          labelText="Danger checked"
+          onChange={() => updateChecked(!checked)}
+        />
 
-  function handleUpdateBlueChecked(event: React.ChangeEvent<HTMLInputElement>) {
-    setCheckedItems([checkedItems[0], event.target.checked]);
-  }
+        <Checkbox
+          checked
+          color={magma.colors.pop02}
+          labelText="Pop checked"
+          onChange={() => updateChecked(!checked)}
+        />
+      </FormGroup>
+      <FormGroup labelText="Disabled">
+        <Checkbox disabled labelText="Disabled checkbox" />
 
-  return (
-    <>
-      <IndeterminateCheckbox
-        onChange={handleUpdateIndeterminateChecked}
-        status={status}
-        labelText="Colors"
-      />
-      <div style={{ marginLeft: magma.spaceScale.spacing08 }}>
         <Checkbox
-          checked={checkedItems[0]}
-          onChange={handleUpdateRedChecked}
-          labelText="Red"
+          defaultChecked={true}
+          disabled
+          id="customId"
+          labelText="Disabled checked checkbox"
         />
+      </FormGroup>
+      <FormGroup labelText="Error">
         <Checkbox
-          checked={checkedItems[1]}
-          onChange={handleUpdateBlueChecked}
-          labelText="Blue"
+          errorMessage="Please check this box"
+          id="customId"
+          labelText="Checkbox with error"
         />
-      </div>
+      </FormGroup>
     </>
   );
 };
+
+export const Inverse = () => {
+  return (
+    <Card isInverse>
+      <CardBody>
+        <FormGroup labelText="Choose One or More">
+          <Checkbox isInverse checked labelText="Uncontrolled checkbox" />
+
+          <Checkbox isInverse labelText="Controlled checkbox" />
+
+          <Checkbox
+            isInverse
+            labelText="Checkbox label is really long and can wrap to multiple lines lorem ipsum dolar sit amet is really long and can wrap to multiple lines"
+          />
+        </FormGroup>
+        <FormGroup labelText="Colors">
+          <Checkbox
+            isInverse
+            checked
+            color={magma.colors.primary}
+            labelText="Primary checked"
+          />
+          <Checkbox
```

Comment:
If we aren't going to support the `color` prop on inverse checkboxes by always making them white, we don't need multiple examples of inverse checkboxes with different colors.  The same would go for the mdx page in the docs.